src/components/atoms/SimpleButton/SimpleButton.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Browsing Storybook I noticed that `SimpleButton` has `cursor: default`. Let's change to `cursor: pointer`, it's more suitable for a button:
![image](https://user-images.githubusercontent.com/8350925/150651781-506dfd58-f179-45a1-8fad-d21d4cb16093.png)